infra/bisector.py

```diff
@@ -18,7 +18,7 @@
 This is done with the following steps:
 
 
-  NOTE: NEEDS TO BE RUN FROM THE OSS-Fuzz HOME directory
+  NOTE: NEEDS TO BE RUN FROM THE OSS-Fuzz HOME directory.
```

Comment:
I get what this says but Home, and in particular HOME refer to a user's home directory, I haven't really seen it used to mean "root" like it is here. Maybe say "root of the OSS-Fuzz source checkout".

Reply:
Done.